llama_stack/providers/tests/inference/groq/test_groq_utils.py

```diff
@@ -416,7 +416,7 @@ class TestConvertStreamChatCompletionResponse:
     async def test_returns_stream(self):
         def chat_completion_stream():
             messages = ["Hello ", "World ", " !"]
-            for i, message in enumerate(messages):
+            for _i, message in enumerate(messages):
```

Comment:
if you don't need `i`, then why not just `for messag in messages:`? what am I missing?

Reply:
🤦🏻